source/tutorials/multi-boot/multi-boot.rst

```diff
@@ -0,0 +1,99 @@
+.. _multi-boot:
+
+Multi-boot |CL-ATTR| and Windows\* OS
+#####################################
+
+This guide shows how to install |CL-ATTR| adjacent to an existing Windows\*
+OS installation. To add |CL| to an existing Windows installation, there are two possible methods:
+
+.. contents::
+   :local:
+   :depth: 1
+
+Method 1: Shrink the Windows partition to make space for |CL|
+*************************************************************
+
+#. Boot up Windows OS 10.
+
+#. Launch the :file:`Disk Management` utility found under
+   :guilabel:`Create and format hard disk partitions`.
+
+#. Right-click the primary Windows partition and select “Shrink Volume...”.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-01.png
+      :width: 75%
+      :alt: Disk Management > Shrink Volume
+
+      Figure 1: Disk Management > Shrink Volume.
+
+#. Shrink the size of the partition by at least the following amount:
+
+   * For :ref:`bare-metal-install-desktop`, allow at least 20GB.
+
+   * For :ref:`bare-metal-install-server`, allow at least 4GB.
+
+#. In this tutorial, We follow :ref:`bare-metal-install-desktop`.
```

Comment:
"In this tutorial, We follow :ref:`bare-metal-install-desktop`." is a little awkward.  Is this the right place for it?